### DISCUSSION TITLE
Does the seawedfs adapt to the ipv6 network?

### DISCUSSION THREAD

**Discussion by jiaqigan** [seaweedfs/seaweedfs#2304] [Q&A] [answered]
Right now I've built a multi server cluster on ipv4 network, however the network is going to update inito ipv6. So I'm wondering the current cluster would be funational as usual. Thanks!

**Comment by chrislusf** [comment, marked as answer]
yes.  https://github.com/chrislusf/seaweedfs/wiki/FAQ#support-ipv6
